capstone-app/src/app/modules/history/wiki-results.service.ts

```diff
@@ -53,17 +53,23 @@ export class WikiResultsService {
       const startIndex = firstPartOfString.indexOf('</h2>', 0);
       let middleOfString = firstPartOfString.substring(startIndex, endIndex);
       const paragraphs = middleOfString.split('<p>');
-      if (paragraphs.length > 6){
-        middleOfString = paragraphs.slice(0,6).join('');
+      console.log(paragraphs);
+      if (paragraphs.length > 9) {
+        if(paragraphs[0].startsWith("</h2")) {
+          middleOfString = paragraphs.slice(1,9).join('');
+        }
```

Comment:
If you want to display the first nine paragraphs and you need to omit the "</h2" paragraph, should we make this slice (1, 10)?

Reply:
Ah, yes. Good point!